### PR TITLE
fix: avatar emotes popup

### PR DESCRIPTION
**Pull request by pbosio**
#898 

- Close popup when animation is triggering using hotkeys
- Close popup by clicking on the background
- Close popup by pressing ESC key

### CODE REVIEW

**Review by RominaMarchetti — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs [hunk 1]
```diff
@@ -32,8 +32,20 @@ private void Awake()
...
+
+        for (int i = 0; i < hideContentButtons.Length; i++)
+        {
+            hideContentButtons[i].onClick.AddListener(HideContent);
+        }
+    }
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape))
```
Comment (pravusjif) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs:45]: This probably should be done using our InputController

Comment (pravusjif) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs]: can this be done using the InputController?

**Review by pravusjif — CHANGES_REQUESTED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs]: > can this be done using the InputController?

Good point. I'll check that out

**Review by sandrade-dcl — APPROVED** [review]
Once the request from Pravus be applied, everything is fine! :)

**Review by pravusjif — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs (hunk 1 above)
Comment (pravusjif) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExpressionsHUD/ExpressionsHUDView.cs]: awsome thx!

**Pull request merged**